misc/emacs-mode/scilla-mode.el

```diff
@@ -233,6 +233,7 @@
 
 (provide 'scilla-mode)
 (add-to-list 'auto-mode-alist '("\\.scilla\\'" . scilla-mode))
+(add-to-list 'auto-mode-alist '("\\.scillib\\'" . scilla-mode))
```

Comment:
README.md needs to be changed accordingly (around line 139).

Reply:
Done.